apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ApplicationInformation/EducationRequirementsDisplay.tsx

```diff
@@ -10,14 +10,119 @@ import { notEmpty } from "@gc-digital-talent/helpers";
 
 import ExperienceTreeItems from "~/components/ExperienceTreeItems/ExperienceTreeItems";
 
+const EducationRequirement_PoolCandidateFragment = graphql(/* GraphQL */ `
+  fragment EducationRequirement_PoolCandidate on PoolCandidate {
+    pool {
+      classification {
+        group
+      }
+    }
+    educationRequirementOption {
+      value
+    }
+    educationRequirementExperiences {
+      id
+      details
+      user {
+        id
+        email
+      }
+      skills {
+        id
+        key
+        name {
+          en
+          fr
+        }
+        category {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        experienceSkillRecord {
+          details
+        }
+      }
+      ... on AwardExperience {
+        title
+        issuedBy
+        awardedDate
+        awardedTo {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        awardedScope {
+          value
+          label {
+            en
+            fr
+          }
+        }
+      }
+      ... on CommunityExperience {
+        title
+        organization
+        project
+        startDate
+        endDate
+      }
+      ... on EducationExperience {
+        institution
+        areaOfStudy
+        thesisTitle
+        startDate
+        endDate
+        type {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        status {
+          value
+          label {
+            en
+            fr
+          }
+        }
+      }
+      ... on PersonalExperience {
+        title
+        description
+        startDate
+        endDate
+      }
+      ... on WorkExperience {
+        role
+        organization
+        division
+        startDate
+        endDate
+      }
+    }
+  }
+`);
```

Comment:
This is admittedly a lot of info. I kind of wish we did not need to query all this just to show the experience they selected for the education requirement :thinking: 

I kinda just ripped this from the career timeline page. Maybe we need to create a fragment for `ExperienceCard` so we know we are only ever querying the required data?